lms/djangoapps/verify_student/models.py

```diff
@@ -721,8 +721,8 @@ def submit(self, copy_id_photo_from=None):
                 self.status = "must_retry"
```

Comment:
@noraiz-anwar Don't we need to cover this part of the error when response is no `ok`